app/views/admin/merchants/index.html.erb

```diff
@@ -1,8 +1,31 @@
 <%= render "/admin/admin_header" %>
 
-<h1>Admin Merchant Index Page</h1>
+<section id="enabled_merchants">
+  <ul>
+    <h4>Enabled Merchants</h4>
+    <% @enabled_merchants.each do |merchant| %>
+      <li><%= link_to "#{merchant.name}", admin_merchant_path(merchant) %>
```

Comment:
just a formatting thought here: it would be ideal if we could get the link and enable/disable button to be next to each other (not on different lines) so it could more closely match the wireframe. I'm not sure if this is possible when using an unorganized list like this? Perhaps consider using a different format? (something other than a list so we could get them next to each other)